modules/810-deckhouse-web/templates/deployment.yaml

```diff
@@ -50,7 +50,7 @@ spec:
       volumes:
         - name: config
           configMap:
-            name: deckhouse-web
+            name: d8-deckhouse-version-info
```

Comment:
810-deckhouse-web reads cm/deckhouse to get values from 'deckhouse.releaseChannel' value. The idea is to create cm in 020-deckhouse module. `helm upgrade` in 810-deckhouse-web without renaming will delete cm/deckhouse-web.

I've tried this locally with simplified chart and also in my testing cluster, everything is fine.
There is [issue](https://github.com/kubernetes-sigs/kind/issues/771) about 'Pod stucking', but it seems irrelevant as cm mounted with 'subpath'.

Reply:
No, this is the real problem. It happens from time to time, and if it does not occur in a single cluster does not mean that it will not happen in other 1000.

Reply:
OK, I've restored template for cm/deckhouse-web with notes about deprecation.